### PR TITLE
fix(trash): accessing from undefined on permanent deleting

### PR DESCRIPTION
**Pull request by jyash97**
## What kind of change does this PR introduce?
Bug Fix

## What is the current behavior?
DeleteSandbox

## What is the new behavior?
LocalDeleteSandbox

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Delete 2-3 sandboxes permanently

## Checklist

- [ ] Documentation - N/A
- [x] Testing 
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3523/commits/7d0188f80f318a1970c2e0cbe7903144e71d698f is at https://pr3523.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Content/Sandboxes/utils.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import { uniqBy } from 'lodash-es';
+import { uniqBy, get } from 'lodash-es';
 import getDefinition from '@codesandbox/common/lib/templates';
 
 export function getPossibleTemplates(sandboxes: any[]) {
   return uniqBy(
     sandboxes.map(x => {
-      const template = getDefinition(x.source.template);
+      const templateId = get(x, 'source.template');
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Dashboard/Content/Sandboxes/utils.ts]: You could also use optional chaining instead.

```suggestion
      const templateId = x.source?.template;
```

File: packages/app/src/app/pages/Dashboard/Content/Sandboxes/utils.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
-import { uniqBy } from 'lodash-es';
+import { uniqBy, get } from 'lodash-es';
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Dashboard/Content/Sandboxes/utils.ts]:
```suggestion
import { uniqBy } from 'lodash-es';
```

**Comment by MichaelDeBoey** [comment]
Ping @jyash97

**Comment by jyash97** [comment]
Hey @MichaelDeBoey sorry for the delay. Thanks for reviewing and suggesting changes :)

**Comment by CompuIves** [comment]
Nice fix! Thank you!

**Pull request merged**